client/src/containers/AddressPage.tsx

```diff
@@ -160,7 +160,16 @@ export default class AddressPage extends Component<AddressPageProps, State> {
               <div className="float-left">
                 <h1 className="primary">
                   <Trans>
-                    PORTFOLIO: Your search address is associated with <u>{assocAddrs.length}</u>{" "}
+                    PORTFOLIO: Your search address is associated with{" "}
+                    <Link
+                      to={routes.portfolio}
+                      tabIndex={this.props.currentTab === 2 ? -1 : 0}
+                      onClick={() => {
+                        window.gtag("event", "portfolio-tab");
```

Comment:
maybe we should name this event something different, like num_addrs_click or something, to differentiate it from someone clicking the portfolio tab. would be interesting to know how many people are clicking that vs the tab

Reply:
good idea, just changed that. that'll be nice to know if it's being used. Originally I was just thinking that I didn't want to mess up the counts of portfolio visits, but we can just remember to add these two tags to get all portfolio tab visits.